### ISSUE TITLE
Fork blocked on loading a repository

### ISSUE THREAD

**Issue by Geobert** [ForkIssues/TrackerWin#419]
Fork 1.38.0.0 on Windows.

I can't load a repository out of the blue. I was working on a project, and commit some files. Then it started to "loading…" and never end.

The repository is https://github.com/cobalt-org/cobalt.rs but if I clone again, it will work.

How can I provide debug information to understand what's happening?

**Comment by DanPristupov** [comment]
Does log file contain any error or warning `%localappdata%\fork\logs\fork.log` ?

**Comment by Geobert** [comment]
```
D 2019-09-02 17:01:43.335 Loading settings.
I 2019-09-02 17:01:43.427 Start ipc server Fork_7072_Pipe_AskPass
I 2019-09-02 17:01:43.427 Start ipc server Fork_7072_Pipe_Default
D 2019-09-02 17:01:43.427 Waiting for next cross-process event '325'
D 2019-09-02 17:01:43.427 Waiting for next cross-process event '325'
I 2019-09-02 17:01:43.537 Fork 1.38.0.0 at C:\Users\Geob\AppData\Local\Fork\app-1.38.0\Fork.exe
I 2019-09-02 17:01:43.537 Git Location: C:\Users\Geob\AppData\Local\Fork\gitInstance\2.20.1\bin\git.exe
I 2019-09-02 17:01:43.537 Log target: AppData log file
I 2019-09-02 17:01:43.686 Last update check: 02/09/2019 16:32:42. Next one: 03/09/2019 16:32:42
I 2019-09-02 17:01:43.929 Restoring 1 sessions.
I 2019-09-02 17:01:44.209 RefreshRepositoryData
I 2019-09-02 17:01:44.209 Main: 246, Window: 559, WindowLoaded: 391, UiReady: 288, Total: 1238
D 2019-09-02 17:01:44.209 cobalt.rs RefreshRepositoryDataGitCommand All
D 2019-09-02 17:01:44.290 Loading cobalt.rs settings.
I 2019-09-02 17:02:21.261 RefreshRepositoryData
D 2019-09-02 17:02:21.277 cobalt.rs RefreshRepositoryDataGitCommand All
I 2019-09-02 17:02:24.583 RefreshRepositoryData
D 2019-09-02 17:02:24.591 cobalt.rs RefreshRepositoryDataGitCommand All
I 2019-09-02 17:02:32.238 RefreshRepositoryData
D 2019-09-02 17:02:32.243 cobalt.rs RefreshRepositoryDataGitCommand All
I 2019-09-02 17:02:43.532 Automatically fetching 'cobalt.rs'
I 2019-09-02 17:02:44.215 RefreshRepositoryData
D 2019-09-02 17:02:44.215 cobalt.rs RefreshRepositoryDataGitCommand All
I 2019-09-02 17:02:44.385 RefreshRepositoryData
D 2019-09-02 17:02:44.385 cobalt.rs RefreshRepositoryDataGitCommand All
```

Blocked on `D 2019-09-02 17:01:44.290 Loading cobalt.rs settings.` 

Have some "Everything is up to date" (from auto fetch I think) but still loading

**Comment by DanPristupov** [comment]
Did you try to restart Fork/restart computer?

**Comment by Geobert** [comment]
restarted fork multiple time, Let me test the reboot (brb)

**Comment by Geobert** [comment]
Rebooting does not help

**Comment by DanPristupov** [comment]
OK, I'm making a debug Fork build right now. Please contact me by email (support@fork.dev) and I'll send it to you.

**Comment by Geobert** [comment]
Thank you Dan for the investigation.
If ever some ppl encounter this before 1.39 is released here is what's happening:
> There was a conflict during reference sorting and local branch ‘master’ conflicted with tag ‘master’.
> If you can, please remove the master tag from command line:
> git tag --delete master

**Comment by DanPristupov** [comment]
I just released the 1.39 update.
